### ISSUE TITLE
duration is always 0 for web recorded media

### ISSUE THREAD

**Issue by nick-delirium** [carsonDB/frameflow#2]
```
const video = await fflow.source(videoBlob());
console.log(video.duration, video.metadata.tracks);
// 0, then two streams, each has 0 as duration
```

which later becomes a problem:

```
const result = await video
      .trim({ start: 5, duration: 5 })
      .exportTo(Blob, { format: "webm" });
```

Will trim from 5th second to end of the video instead of next 5 seconds, ignoring duration because [here](https://github.com/carsonDB/frameflow/blob/main/src/ts/filters.ts#L26) it will be `Math.min(5 + 0, 0 + 0)`

Can you point to any synthetic solution for now on? Otherwise, library is fantastic, thank you.

**Comment by nick-delirium** [comment]
Actually I'm pretty sure this is the same issue that's causing 0 duration mediaRecorder webms in Chrome browser. There are [some libraries that fix it](https://github.com/yusitnikov/fix-webm-duration), that this just causes more issues with this library, although it parses duration correctly after fixing

![image](https://github.com/carsonDB/frameflow/assets/23126999/74988a4b-f124-44a8-a3c7-7a914f488e44)

**Comment by carsonDB** [comment]
Yes, you are right. It's Chrome's problem. In this library, frameflow can directly receive stream of frames and encode to mp4 file. Like this example: https://github.com/carsonDB/frameflow/blob/main/examples/browser/codec.html#L51
